pkg/sql/resolver.go

```diff
@@ -1077,9 +1074,6 @@ type internalLookupCtx struct {
 	tbIDs       []descpb.ID
 	typDescs    map[descpb.ID]catalog.TypeDescriptor
 	typIDs      []descpb.ID
-
-	// fallback is utilized in GetDesc and GetNamespaceEntry.
-	fallback catalog.DescGetter
```

Comment:
As it turns out, this wasn't being used! Good riddance.

Reply:
Are you certain? In my head, we needed it for cross-database foreign key references in some virtual tables. I hope you're right. That'd be great. I suppose what could be going on is that we're just always fetching all of the descriptors in those cases.

Reply:
I believe that that's the case, yes.